subscription.go

```diff
@@ -230,7 +227,7 @@ func (s *Subscription) subscribeLogs() {
 			s.done <- true
 			return
 
-		case err = <-subErrChan:
+		case err = <-errch:
```

Comment:
why does it return the line 238?

Reply:
to terminate this goroutine

Reply:
But this part
```
if isRetryable(err) {
    s.subscribeLogs()
}
```
blocks the `return` statement

Reply:
nice catch! fixed it! thanks